cli/file_copy_getter.go

```diff
@@ -30,7 +34,7 @@ func (g *FileCopyGetter) Get(dst string, u *url.URL) error {
 		return fmt.Errorf("source path must be a directory")
 	}
 
-	return util.CopyFolderContents(path, dst)
+	return util.CopyFolderContents(path, dst, SOURCE_MANIFEST_NAME)
```

Comment:
Should this be using a different constant than `MODULE_MANIFEST_NAME`?

Reply:
Yes, because if a local source is used, then two separate manifest files need to be created: one for the local source, and the other for the terragrunt module directory.

Reply:
Ohhh, I gotcha. I'll update one comment to make that clearer.